src/profile_manager/forms.py

```diff
@@ -31,6 +32,15 @@ def __init__(self, *args, **kwargs):
 
         self.fields['email'].initial = self.instance.user.email
 
+        # gets value from deck's custom_name_for_student field in siteconfig.models
+        custom_student_name = SiteConfig.objects.get().custom_name_for_student
+        # sets visible_to_other_students field name and help text on student profile settings page
+        self.fields['visible_to_other_students'].label = f"Visible to other {custom_student_name.lower()}s"
+        # setting a with variable isn't ideal but linebreaking with a backslash breaks tests that parse for the exact help text.
```

Comment:
Setting a `with` variable where? I don't see a with.

Reply:
Comment text reworded to resolve grammatical error and better represent referenced code's intention

` # setting help text with a variable in two steps isn't ideal but linebreaking with a backslash breaks tests that parse for exact help text.`